### PR TITLE
phoedf changes

### PR DESCRIPTION
**Pull request by brryan**
## PR Summary

Some quantities in `phoedf` weren't quite complete.

## PR Checklist

- [x] Adds a test for any bugs fixed. Adds tests for new features.
- [x] Format your changes by calling `scripts/bash/format.sh`.
- [x] Explain what you did.
- [ ] Merge https://github.com/parthenon-hpc-lab/parthenon/pull/795 and then update the `parthenon` submodule here.

### CODE REVIEW

**Review by Yurlungur — APPROVED** [review]

File: scripts/python/plot_torus_3d.py [hunk 1]
```diff
@@ -31,14 +31,11 @@
 import phoebus_utils
 from phoedf import phoedf
 
-def plot_frame(ifname, fname, savefig, geomfile=None, rlim=40, coords='cartesian', draw_bh=True):
-  print(fname)
+def plot_frame_from_phoedf(dfile, ifname, savefig, geomfile=None, rlim=40, coords='cartesian', draw_bh=True, custom_name=None):
```
Comment (Yurlungur) [inline scripts/python/plot_torus_3d.py:34]: this is long. Can you split it across multiple lines.

File: scripts/python/plot_torus_3d.py [hunk 2]
```diff
@@ -277,10 +274,20 @@ def plot_frame(ifname, fname, savefig, geomfile=None, rlim=40, coords='cartesian
...
 
-  savename = str(ifname).rjust(5,"0") + '.png'
+  if custom_name is not None:
+    savename = custom_name
+  else:
+    savename = str(ifname).rjust(5,"0") + '.png'
   plt.savefig(savename, dpi=300, bbox_inches='tight')
   plt.close(fig)
 
+def plot_frame(ifname, fname, savefig, geomfile=None, rlim=40, coords='cartesian', draw_bh=True, custom_name = None):
```
Comment (Yurlungur) [inline scripts/python/plot_torus_3d.py:284]: this too

File: scripts/python/torus_analyzer.py [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+from enum import Enum
+from multiprocessing import Pool
+from phoebus_constants import cgs, scalefree
+import phoebus_utils
+from phoedf import phoedf
+
+from get_torus_fluxes import get_torus_fluxes
+from plot_torus_3d import plot_frame_from_phoedf
+
+# Script to perform complete analysis of an output directory
```
Comment (Yurlungur) [inline scripts/python/torus_analyzer.py:37]: this is pretty nice

**Review by Yurlungur — APPROVED** [review]

File: scripts/python/torus_analyzer.py (hunk 3 above)
Comment (brryan) [reply scripts/python/torus_analyzer.py:37]: Yeah you can just call this file a bunch for different output directories in a slurm submission script and get all your analysis done in one go.

File: scripts/python/plot_torus_3d.py (hunk 1 above)
Comment (brryan) [reply scripts/python/plot_torus_3d.py:34]: Done (I used `black`; maybe we should add that to `scripts/bash/format.sh` if `black` exists on the system?)

Comment (Yurlungur) [reply scripts/python/plot_torus_3d.py]: that's a good idea.

**Comment by Yurlungur** [comment]
@brryan is this ready for merge? I see the parthenon PR is still open.

**Comment by brryan** [comment]
@Yurlungur yep that parthenon PR is merged now. I'll merge this after checks pass (had to updated `external/parthenon`).

**Pull request merged**